contrib/ohi-release/README.md

```diff
@@ -0,0 +1,63 @@
+# 🛠️ `ohi-release`
+
+This is a wrapper of all the steps needed to update the changelog and render a snippet to be ready to be used as a release message. This contribution also includes a script that allow to replicate what this action does locally.
+
+## Example Usage
+
+```yaml
+- uses: newrelic/release-toolkit/contrib/ohi-release@v1
+  id: release
+- name: Commit updated changelog
+  run: |
+    git add CHANGELOG.md
+    git commit -m "Update changelog with changes from ${{ steps.release.outputs.next-version }}"
+    git push -u origin main
+    gh release create ${{ steps.release.outputs.release-title }} --target $(git rev-parse HEAD) --notes-file CHANGELOG.partial.md
+```
+
+## Parameters
+
+All parameters are optional:
+  * `excluded-dirs` exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".
```

Comment:
Why not adding also the included-dirs flag? At the moment we don't have any case where we want to include only one folder instead of excluding tones of them, but it could happen.

Reply:
That is the point I did not add it. I did not want to expose all the flags possible for all the commands that this wrapper runs until it is needed.

It you can barely thing a single use case I'll be happy to add it :)